src/main/scala/io/pinecone/spark/pinecone/package.scala

```diff
@@ -15,26 +15,29 @@ package object pinecone {
       .add("namespace", StringType)
       .add("values", ArrayType(FloatType))
       .add("metadata", StringType)
+      .add("sparse_id", ArrayType(IntegerType, containsNull = true), nullable = true)
```

Comment:
Added sparse_id and sparse_values to the input file. It can null as well since dense vectors are default at Pinecone